build/dev/run_locally.sh

```diff
@@ -5,8 +5,8 @@ set -eo pipefail
 # This script will deploy a standalone DSS instance with docker-compose using
 # images from Docker Hub.
 
-if [[ -z $(command -v docker-compose) ]]; then
-  echo "docker-compose is required but not installed.  Visit https://docs.docker.com/compose/install/ to install."
+if [[ -z $(command -v docker compose) ]]; then
+  echo "docker compose is required but not installed.  Visit https://docs.docker.com/compose/install/ to install."
```

Comment:
If this check fails, docker isn't installed, not docker-compose:

```suggestion
  echo "docker is required but not installed.  Visit https://docs.docker.com/get-docker to install."
```